src/sage/schemes/curves/zariski_vankampen.py

```diff
@@ -980,7 +1461,13 @@ def fundamental_group(f, simplified=True, projective=False):
       of the curve will be computed, otherwise, the fundamental group of
       the complement in the affine plane will be computed
 
-    If ``simplified`` is ``False``, a Zariski-VanKampen presentation
+    - ``puiseux`` -- boolean (default: ``False``); if set to ``True``,
+      ``simplified`` is set to ``False``, and
```

Comment:
a presentation... is computed, `simplified`is ignored.

Reply:
Done, hopefully